build.sh

```diff
@@ -8,6 +8,7 @@ BUILD_DIR="${CUR_DIR}/.build_${ARCH}"
 DEBIAN_VERSION="${DEBIAN_VERSION:-jessie}"
```

Comment:
abusing this spot for a small note;
don't make commit messages in the past tence 'e.g. Make use of' rather then 'made use of'. This commit performs an action, it will do it when applying the commit.
Its just semantics so don't worry about changing it, just something for the future to keep in mind